version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "2.0.0-SNAPSHOT"
+ThisBuild / version := "2.0.0-alpha.18"
```

Comment:
@ryle-goehausen I'm not sure. I thought this would be a branch that @IPadawans would download to run on the two environments. We can cherry-pick the changes to develop if we're keeping it.

Reply:
The tests currently will fail if we change this file for any value except 2.0.0-SNAPSHOT. I need to fix this after some other priorities

Reply:
I pushed `-SNAPSHOT`